pkg/virt-launcher/virtwrap/device/hostdevice/hostdev.go

```diff
@@ -83,8 +86,13 @@ func createHostDevices(hostDevicesData []HostDeviceMetaData, addrPool AddressPoo
 			}
 		}
 		hostDevices = append(hostDevices, *hostDevice)
-		log.Log.Infof("host-device created: %s", address)
+		hostDevicesAddresses = append(hostDevicesAddresses, address)
 	}
+
+	if len(hostDevices) > 0 {
+		log.Log.Infof("host-devices created: %v", hostDevicesAddresses)
```

Comment:
While I agree with the idea compacting the logs into a single one, I think you should also consider the readability of the addresses in the single log. While verbose, the current implementation with multiple logs separates the addresses by line, making them easier to read. This single log implementation would display the array of addresses in its default format of `[address1 address2 address3]` with no clear visual separation of the different addresses; this could become difficult to read if multiple devices are created. 

It is worth considering modifying the format of the list of addresses in the consolidated log to improve the readability of the log.

Reply:
Thanks for the feedback, I've added separating addresses with an additional comma